src/call_reply.h

```diff
@@ -0,0 +1,28 @@
+#ifndef SRC_CALL_REPLY_H_
+#define SRC_CALL_REPLY_H_
+
+#include "reply_parser.h"
+
+typedef struct CallReply CallReply;
+
+CallReply* callReplyCreate(sds reply, void* private_data);
```

Comment:
Please try to follow Redis code style, i.e. `CallReply *callReply` etc.

Reply:
Cover here: https://github.com/redis/redis/pull/9202/commits/2b6cfd87189b0ef94b2280e220618140b6824818
Thanks @yossigo